srcpkgs/nordselect/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'nordselect'
+pkgname=nordselect
+version=1.4.2
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config liboping-devel"
+makedepends="openssl-devel liboping-devel"
+depends="openssl liboping"
```

Comment:
It detects both dependencies automatically, see the build log:

```
    SONAME: liboping.so.0 <-> liboping>=1.8.0_1
   SONAME: libssl.so.1.1 <-> libssl1.1>=1.1.1f_1
   SONAME: libcrypto.so.1.1 <-> libcrypto1.1>=1.1.1f_1
   SONAME: libgcc_s.so.1 <-> libgcc>=4.4.0_1
   SONAME: libpthread.so.0 <-> glibc>=2.32_1
   SONAME: libm.so.6 <-> glibc>=2.32_1
   SONAME: libdl.so.2 <-> glibc>=2.32_1
   SONAME: libc.so.6 <-> glibc>=2.32_1
   SONAME: ld-linux.so.2 <-> glibc>=2.32_1
```

`openssl` can be added if it actually executes the `openssl` program for some functionality.

Otherwise, just remove `depends`.

Reply:
`depends` is removed now